tests/ui-toml/invalid_min_rust_version/invalid_min_rust_version.stderr

```diff
@@ -1,4 +1,4 @@
-error: error reading Clippy's configuration file `$DIR/clippy.toml`: `invalid.version` is not a valid Rust version
+error: `invalid.version` is not a valid Rust version
```

Comment:
I would keep the massage here, that this configuration comes from the configuration file. The filename doesn't have to be included in the error message though.